index.js

```diff
@@ -36,12 +36,16 @@ module.exports = {
         'jest/globals': true,
       },
       rules: {
+        'jest/no-alias-methods': 'warn',
```

Comment:
damn, I guess I like the aliased methods better :D

Reply:
I think it makes sense? The justification in the docs of the rule makes sense to me, +1 for consistency.

If the aliased names are better, maybe we should swap them in Jest? :D

Reply:
I like them because they're shorter, but don't care too much because Flow autocompletes it for me. Consistency matters though. I wish it was autofixable though!

Reply:
PR welcome? 😀

Reply:
Wait, it _is_ fixable: https://github.com/jest-community/eslint-plugin-jest/blob/db0fdd835050bbe6e96d4ce79706d45a34760bad/rules/no-alias-methods.js#L49-L51

Reply:
I was just about to write that!

Reply:
Updated the readme to include the badge. Also updated the readme in this PR to reflect which rules are recommended, I forgot that

Reply:
How about adding `prefer-to-contain`, `prefer-to-be-undefined` and `prefer-to-be-null`? They're pretty useful and all fixable

Reply:
What about my suggestion in the OP?

>I removed the stylistic matcher stuff. We could export something like `plugin:jest/stylistic`

Reply:
Makes sense. Maybe jest/style would be easier to type

Reply:
`jest/style` sounds good to me too.

Reply:
done!